Yoast/wordpress-seo
# Rtl fixes.

## Context


* Fixes a couple of RTL issues on the release branch for indexables.

## Summary


This PR can be summarized in the following changelog entry:

* Fixes a bug where the Helpscout beacon on settings pages was aligned on the right of the settings pages.
* Fixes a bug where UTF-8 encoded urls were not always readable in the snippet preview.
* Reimplements some changes from #14511. Canonical, sameAs schema and Opengraph Url output is made readable again.

## Test instructions

This PR can be tested by following these steps:

* Set your site language to Hebrew and add a post with a Hebrew url. Publish and refresh the editor. See the output for the slug in the snippet editor is now encoded. This PR fixes that.
* Add a Hebrew or Arabic canonical url. See how those urls are output encoded on the frontend on the release branch. This PR fixes that.
* Set your site language to Hebrew and go to our settings. See the Helpscout beacon is overlaying the menu on the righthand side. This PR puts it on the left.

## UI changes

## Quality assurance
